src/request.ts

```diff
@@ -42,6 +43,11 @@ Object.defineProperty(request, 'setDefaultProvider', {
 
 providerRegistry.setDefaultProvider(xhr);
 
+if (!has('host-browser')) {
```

Comment:
While this is probably fine, I'd prefer an explicit feature test for Node if possible. I know that we currently only focus on browsers or Node, but a lot can change in the future.